packages/syft/src/syft/service/project/project.py

```diff
@@ -1105,10 +1109,29 @@ def exchange_routes(
 
         return SyftSuccess(message="Successfully Exchaged Routes")
 
+    def create_request(self, obj: SubmitUserCode, client: SyftClient):
```

Comment:
should we rename it to something related to user code maybe `create_code_request` ?